gitversion/dobi.yaml

```diff
@@ -5,7 +5,7 @@
 image=image-gitversion:
   image: elbb/bb-gitversion
   context: gitversion
-  tags: ["{env.GitVersion_MajorMinorPatch}"]
+  tags: ["{env.GitVersionDocker}"]
```

Comment:
Please replace the 'Docker' with 'Branch'

Reply:
done